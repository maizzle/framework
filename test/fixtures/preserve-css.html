<!doctype html>
<html>
  <head>
    <style postcss preserve>
      div {
        @apply uppercase;
      }

      u + #body a {
        color: inherit;
        text-decoration: none;
        font-size: inherit;
        font-weight: inherit;
        line-height: inherit;
      }
    </style>
  </head>
  <body>
    <div>test</div>
  </body>
</html>
