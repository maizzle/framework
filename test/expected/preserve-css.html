<!doctype html>
<html>
  <head>
    <style data-embed="">
      div {
        text-transform: uppercase;
}

      u + #body a {
        color: inherit;
        text-decoration: none;
        font-size: inherit;
        font-weight: inherit;
        line-height: inherit;
      }
    </style>
  </head>
  <body>
    <div>test</div>
  </body>
</html>
